### PR TITLE
[Doc]Document all the currently supported SQL operation for postgresql connector

### LINKED ISSUE #22245: [Doc] Document all the currently supported SQL operation for postgresql connector
Document all the currently supported SQL operations for postgresql connector along with an example similar to https://prestodb.io/docs/current/connector/iceberg.html#sql-support

## Expected Behavior or Use Case
To provide more concrete details to users.
Once we have documentation for postgresql connector, we can document similar supported SQL operations for other JDBC connectors as well.

## Presto Component, Service, or Connector
Presto Docs(postgresql)

## Possible Implementation
[](https://prestodb.io/docs/current/connector/iceberg.html#sql-support)

## Example Screenshots (if appropriate):
presently only :
Screenshot 2024-03-20 at 9 24 49 AM

## Context

To provide more concrete details to users

### PR DESCRIPTION
**Pull request by Akanksha-kedia**
Document all the currently supported SQL operations for postgresql connector along with an example similar to https://prestodb.io/docs/current/connector/iceberg.html#sql-support

## Expected Behavior or Use Case
To provide more concrete details to users.
Once we have documentation for postgresql connector, we can document similar supported SQL operations for other JDBC connectors as well.

Fixes #22245

## Presto Component, Service, or Connector
Presto Docs(postgresql)

## Possible Implementation
[](https://prestodb.io/docs/current/connector/iceberg.html#sql-support)

## Example Screenshots (if appropriate):
presently only :
Screenshot 2024-03-20 at 9 24 49 AM

## Context

To provide more concrete details to users

## Impact
No impact

## Test Plan
manually built the doc and using mvn install and generated html .

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```
 
 Testing ss:
 
Screenshot 2024-03-20 at 10 17 13 AM
Screenshot 2024-03-20 at 10 17 21 AM
Screenshot 2024-03-20 at 10 17 29 AM

### CODE REVIEW

**Review by steveburnett — CHANGES_REQUESTED** [review]
In the cited [Iceberg Connector](https://prestodb.io/docs/current/connector/iceberg.html#sql-support) page, the topic is titled "SQL Support". In this draft you call the topic "PostgreSQL Connector Support". Why the difference? "PostgreSQL Connector Support" of what?

Also consider the review I did yesterday for your https://github.com/prestodb/presto/pull/21949 and apply the suggestions I made there, as appropriate, to this draft.

**Comment by Akanksha-kedia** [comment]
@steveburnett done.

File: presto-docs/src/main/sphinx/connector/postgresql.rst [hunk 1]
```diff
@@ -192,16 +192,108 @@ only unsupported data types, Presto returns an error similar to the following ex
...
+
+    TRUNCATE TABLE postgresql.public.renamed_table;
+
 PostgreSQL Connector Limitations
 --------------------------------
 
-The following SQL statements are not yet supported:
+The following SQL statements are not supported:
 
+* :doc:`/sql/alter-table` (ALTER COLUMN TYPE)
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/postgresql.rst:289]: I am not sure I understand the meaning of `(ALTER COLUMN TYPE)` here. The capitalization suggests that when I follow the link to the ALTER TABLE page, I might find ALTER COLUMN TYPE but those words are not there. 
Can you explain?

File: presto-docs/src/main/sphinx/connector/postgresql.rst [hunk 2]
```diff
@@ -192,16 +192,108 @@ only unsupported data types, Presto returns an error similar to the following ex
...
+* :doc:`/sql/create-schema`
+* :doc:`/sql/create-table` (WITH PRIMARY KEY)
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/postgresql.rst:291]: I also do not understand what you mean by the words `(WITH PRIMARY KEY)` here. Can you explain?

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks for the revision and fixes! I had some suggestions to the text, and a couple of questions I hope you can help with.

**Comment by Akanksha-kedia** [comment]
i have added scenario and removed from not supported.

**Comment by Akanksha-kedia** [comment]
Screenshot 2024-03-23 at 2 25 35 PM
@steveburnett please review.

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

**Comment by steveburnett** [comment]
Please add a release note of 
```
== NO RELEASE NOTE ==
```

**Comment by Akanksha-kedia** [comment]
@tdcmeehan please review and help to merge.

**Pull request merged**